compiler/src/dotty/tools/backend/sjs/JSPrimitives.scala

```diff
@@ -24,8 +24,9 @@ object JSPrimitives {
   final val JS_NATIVE = TYPEOF + 1  // js.native. Marker method. Fails if tried to be emitted.
 
   final val UNITVAL = JS_NATIVE + 1 // () value, which is undefined
+  final val UNITTYPE = UNITVAL + 1 // () type
```

Comment:
This seems in fact unused?

Reply:
It seems so - probably doesn't need to be defined the way UNITVAL is. Removed.